### PR TITLE
Add monitoring script for checking blank screen

### PR DESCRIPTION
**Pull request by XiaoxuHU**
## Overview
This PR is for adding monitoring.js script to monitoring blank screen issue for our Angular project 

## Related Work Item

[WI 2953 Set up more alerts on the SCIFrame blade to catch scenarios when no detector is found, or no expected parameters are present](https://dev.azure.com/app-service-diagnostics-portal/app-service-diagnostics-portal/_workitems/edit/2953)
## Type of change

- [X] New feature (non-breaking change which adds functionality)

## Solution description

Added monitoring.js scrpit, it will be running once scripted ready. For checking <sc-app> which it a tag created by Angular to make sure our page is not blank. 

If it is a blank screen we will log "PortalBlankPage" event into Kusto table so to get alert based on this event

### CODE REVIEW

File: AngularApp/projects/app-service-diagnostics/src/assets/scripts/monitoring.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+    postMessage(verb, data) {
+        if (LoggingUtilities.inIFrame()) {
+            let dataString = data;
+            try {
+                var dataJsonObject = data === null ? {} : JSON.parse(data);
+                const dataObjectWithEventType = {
+                    signature: LoggingUtilities.iFrameSignature,
+                    eventType: verb,
+                    ...dataJsonObject,
```
Comment (nmallick1) [inline AngularApp/projects/app-service-diagnostics/src/assets/scripts/monitoring.js:15]: Is the spread syntax supported across all browsers that end up hitting us?

Comment (XiaoxuHU) [reply AngularApp/projects/app-service-diagnostics/src/assets/scripts/monitoring.js:15]: I can see spread syntax is supporting for all browsers

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

image

Comment (nmallick1) [reply AngularApp/projects/app-service-diagnostics/src/assets/scripts/monitoring.js:15]: IE 11 does not support spread, do we have any users hitting us with IE 11?

Comment (XiaoxuHU) [reply AngularApp/projects/app-service-diagnostics/src/assets/scripts/monitoring.js:15]: I believe Azure portal already end of supporting IE 

https://azure.microsoft.com/en-us/updates/azure-portal-to-end-support-for-internet-explorer-11-on-march-31-2021/

**Review by nmallick1 — APPROVED** [review]

**Pull request merged**